src/libraries/System.Formats.Asn1/README.md

```diff
@@ -0,0 +1,16 @@
+# System.Formats.Asn1
+
+This assembly provides support for reading and writing values encoded under the [ITU X.690](https://www.itu.int/rec/T-REC-X.690) Basic Encoding Rules (BER), Canonical Encoding Rules (CER), and Distinguished Encoding Rules (DER) encodings of data types defined in [ITU X.680 Abstract Syntax Notation One](https://www.itu.int/rec/T-REC-X.680) (ASN.1).
+
+The primary types in this assembly are [AsnReader](https://learn.microsoft.com/en-us/dotnet/api/system.formats.asn1.asnreader) and [AsnWriter](https://learn.microsoft.com/en-us/dotnet/api/system.formats.asn1.asnwriter).
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/system.formats.asn1
+
+## Contribution Bar
+
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../../libraries/README.md#secondary-bars)
+
+## Deployment
+
+The library is shipped as part of the .NET shared framework and as a [NuGet package](https://www.nuget.org/packages/System.Formats.Asn1). Daily builds of the NuGet package can be obtained following [these instructions](../../../docs/project/dogfooding.md#obtaining-daily-builds-of-nuget-packages).
```

Comment:
I am not sure if we should mention daily builds in README.md. I am just afraid that it might backfire on us (users taking a dependency on something that is going to change in the future or hasn't been tested for all possible scenarios yet). I would remove it from this file and expect that those who for some reason need a daily build are going to do the search and find it.

```suggestion
The library is shipped as part of the .NET shared framework and as a [NuGet package](https://www.nuget.org/packages/System.Formats.Asn1).
```

Reply:
I got that from somewhere else, but have noticed that it's not consistent, so happy to remove.